customer-stack/customer-stack.yml

```diff
@@ -65,6 +73,11 @@ Conditions:
     - !Equals
       - !Ref kmsKeyArn
       - ""
+  shouldCreateRegionalGateway:
+    !Equals [!Ref apiGatewayType, "REGIONAL"]
+  isVPCConfigNotPresent: !And
```

Comment:
Want to confirm - what if a customer only specifies 1 of vpcSubnetIds or vpcSecurityGroupIds? Would we then consider that "VPC config IS present"? Or should both be present?

Reply:
It should be Or. Thanks for pointing out. I have changed the condition name to `isVPCConfigPresent ` and updated the cfn template to accommodate it .